src/NServiceBus.AcceptanceTests/Mutators/When_outgoing_mutator_replaces_instance.cs

```diff
@@ -8,18 +8,17 @@
 
     public class When_outgoing_mutator_replaces_instance : NServiceBusAcceptanceTest
     {
+        static Context testContext = new Context();
```

Comment:
See comment above. Why are you no longer using injection or getting the ATT context over the mutator context?